resource/history/magento/project-community-edition/2.4.7-p3.json

```diff
@@ -0,0 +1,84 @@
+{
+    "name": "magento/project-community-edition",
```

Comment:
Is this the correct file/contents? This differs significantly from the past release files for 2.4.7 in this folder, as well as the others in this folder for this update.

See EG. https://github.com/rhoerr/mage-os.generate-mirror-repo-js/blob/d60a663a5635cdf7be33614b762e2e43bad8d282/resource/history/magento/project-community-edition/2.4.7-p2.json

Reply:
Good catch! Thank you! 💪